### ISSUE TITLE
Selection layout section "main" not included

### ISSUE THREAD

**Issue by operatorone** [inspiredminds/contao-content-templates#3]
Hi, I am trying to include a new article template and can't choose "Hauptspalte" in the select menu "Anzeigen in". Only "Unbekannte Option: main" is displayed.

https://share.cleanshot.com/jy0j7lyXYlZ5fg3886VV

**Comment by fritzmg** [comment]
Yes, articles from content templates aren't associated with a regular Contao page and thus are also not associated with a particular page layout, where the layout sections are defined. I have removed the selection generally for now.
